lib/components/AppCatalog/AppCatalog.tsx

```diff
@@ -98,7 +101,11 @@ const AppCatalog: React.FC<AppCatalogProps> = ({ appsToDisplay }) => {
 
   const appCatalogEntries = appsToDisplay.map((appDetails, index) => {
     return (
-      <Grid item key={`linkToApp${index}`} md={4}>
+      <Grid
+        item
+        key={`linkToApp${index}`}
+        md={catalogMode && catalogMode === 'publisher' ? 'auto' : 4}
```

Comment:
It isn't intended, but here's the thing:

- If I give `md` a value of 3 when `catalogMode === 'publisher'` is true so as to match the designs, the screen behaves as shown in the image. Unfortunately, I did not manage to get around this issue unless I gave `md` a value of `auto`.

<img width="297" alt="Screenshot 2021-07-06 at 15 25 25" src="https://user-images.githubusercontent.com/67902809/124619392-868ca600-de70-11eb-95df-e69f10259d18.png">

Reply:
Why md={3} where does the 3 come from?

Reply:
3 would make for 3 column-wide cards, allowing for 4 cards per row.

In the Marketplace, we have an md of 4 because we have 4-column wide cards, making for 3 cards per row.

Reply:
For posterity:

Turns out, the issue of the above image is due to lack of a fixed width for cards.